src/components/HomeScreen/HomeScreen.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Container, Row, Col } from 'react-bootstrap';
+
+import NavBar from '../NavBar/NavBar';
+import Welcome from '../Welcome/Welcome';
+import RegistrationForm from '../RegistrationForm/RegistrationForm';
+let HomeScreen = () => {
```

Comment:
const & let both have different usage/scope. please update it with const.

Reply:
fixed